photon-api/src/main/scala/com/linkedin/photon/ml/hyperparameter/tuner/HyperparameterTunerFactory.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.hyperparameter.tuner
+
+import com.linkedin.photon.ml.HyperparameterTunerName.{ATLAS, DUMMY, HyperparameterTunerName}
+
+object HyperparameterTunerFactory {
+
+  val DUMMY_TUNER = "com.linkedin.photon.ml.hyperparameter.tuner.DummyTuner"
+  val ATLAS_TUNER = "com.linkedin.photon.ml.hyperparameter.tuner.AtlasTuner"
```

Comment:
Are both of these moving to Atlas? In either case, let's change the string to " com.linkedin.atlas.tuner.AtlasTuner"

Reply:
Since it is not moved yet, the test with com.linkedin.atlas.tuner.AtlasTuner will report an error message. Please see the test result with invalid tuner in li-photon-plugin.

Reply:
Let's put a comment here with a brief explanation of why it's arranged this way, and that the atlas-specific code will be removed.